public/css/common.less

```diff
@@ -328,3 +332,93 @@ div.show-more {
 .servicegropup-list > col.object-statistics-graph {
   padding-right: .25em;
 }
+
+.comment-popup {
+  font-size: 1.2em;
```

Comment:
This results for me in 12.3408px. Make it so that it's exactly 12px.

Reply:
I did it manually, it took a little while, but I fixed it. Is there an easy way to do this? The result of the Google converter was not correct.

Reply:
`1em` are `12px` at the default font size in your scope. For the footer the font-size is reduced to `0.857em`, that are 85,7% of `1em` or `12px`. To get the font-size back to `12px` in relation to `0.857em`, the font-size has to be `1.166861144em` (1 / 0.857).

Since browsers tend to optimize fractions in numbers differently, we usually place the calculation inside the stylesheet to also emphasize what is calculated there: `font-size: 1/.857em; // default font size / footer font size = 12px`

Reply:
Thanks for the explanation 😊  .